File: 3-4주차/B11399.java
import java.util.*;

public class B11399 {
    public static void main(String args[]){
        Scanner in = new Scanner(System.in);

        int N = in.nextInt();
        int[] arr = new int[N];

        for (int i = 0; i < N; i++) {
        arr[i] = in.nextInt();
        }
        
        in.close();
        Arrays.sort(arr);

        int sum = 0, wtime = 0;
        for(int i :arr){
            sum +=  wtime + i;
            wtime += i;
        }

        System.out.print(sum);
    }
}

File: 3-4주차/B11399.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split('\n');

const N = Number(input[0]);
const arr = input[1].split(' ').map(Number);

arr.sort((a, b) => a - b);

let sum = 0, wtime = 0;
for (const i of arr) {
    sum += wtime + i;
    wtime += i;
}

console.log(sum);
